react/react-state-2_table-reservation/src/App.jsx

```diff
@@ -1,12 +1,24 @@
+import { useState } from "react";
 import Counter from "./components/Counter";
 import "./App.css";
 
 export default function App() {
+  const [people, setPeople] = useState(0);
+
+  function peopleIncrement() {
+    setPeople(people + 1);
+  }
+  function peopleDecrement() {
+    if (people > 1) {
```

Comment:
Here you could use the max function. The outcome is the same, but a bit more... elegant... hehe

setPeople(Math.max(0, people - 1));

Basically, get the bigger value, 0 or the result of people - 1